best_practices/production_deployments/plugins/PINNING_VERSIONS.md

```diff
@@ -0,0 +1,77 @@
+# Pinning Versions
+
+## How we version plugins
+
+Plugins follow [Semantic Versioning (semver)](https://semver.org/spec/v2.0.0.html), which requires breaking channels to be communicated through the release version number. Please see [this documentation](https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md) for an in-depth look at how we do so.
+
+## No YOLOver
+
+In production, pinning of plugin versions is an absolute must do for stability of your monitoring infrastructure. Said another way, you should never use unpinned versions of plugins when you install them. This practice is sometimes referred to as `YOLOver` since the only guarantee is that you will break at some point in the future.
+
+Chances are you don't realize you are following this pattern. What this really means is that at install time you are not providing any guidelines on the version to install which will pick up the latest version, even across major versions, which will pretty much always include some form of [breaking changes](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#the-breaking-change). **Keep reading on to verify you're not in this dangerous situation!**
+
```

Comment:
I love the build up to here, but I think the natural next step is to tell people (1) How to verify if they're in YOLO land and (2) How to do it correctly.

Reply:
I don't understand which of the following fits which of those two. It'd be great to give that 1-2 punch here.

Reply:
Awesome, I'm with you on a bit of clarification and then 🚢